tests/functional/modules/test_zos_lineinfile_func.py

```diff
@@ -515,6 +515,28 @@ def test_uss_line_replace_quoted_not_escaped(ansible_zos_module):
     finally:
         remove_uss_environment(ansible_zos_module)
 
+@pytest.mark.uss
+def test_uss_line_does_not_insert_repeated(ansible_zos_module):
+    hosts = ansible_zos_module
+    params = dict(path="", line='ZOAU_ROOT=/mvsutil-develop_dsed', state="present")
+    full_path = TEST_FOLDER_LINEINFILE + inspect.stack()[0][3]
+    try:
+        hosts.all.shell(cmd="mkdir -p {0}".format(TEST_FOLDER_LINEINFILE))
+        hosts.all.file(path=full_path, state="touch")
+        params["path"] = full_path
+        results = hosts.all.zos_lineinfile(**params)
+        for result in results.contacted.values():
+            print(result)
+            assert result.get("changed") == 1
+        results = hosts.all.zos_lineinfile(**params)
+        for result in results.contacted.values():
+            print(result)
+            assert result.get("changed") == 1
+        results = hosts.all.shell(cmd="cat {0}".format(params["path"]))
+        for result in results.contacted.values():
+            assert result.get("stdout") == 'ZOAU_ROOT=/mvsutil-develop_dsed'
```

Comment:
All fair points, it happens to be that the `dev` branch this will merge into will be 1.8.0-beta.1 at the end of the quarter and will increase the minimum version of zoau to 1.2.4 not just for this reason but also because we want to pull in DISP=SHR from `dcp` and program_name from `jls`, my comment on the zoau version, i have not reviewed any code and will wait till after the requested changes are made now to avoid any further confusion.